kafka-client-lib/src/main/java/io/jenkins/plugins/remotingkafka/builder/KafkaClassicCommandTransportBuilder.java

```diff
@@ -86,27 +87,27 @@ public KafkaClassicCommandTransportBuilder withConsumerPartition(int consumerPar
         return this;
     }
 
-    public KafkaClassicCommandTransport build() {
+    public KafkaClassicCommandTransport build() throws RemotingKafkaTransportException {
```

Comment:
wold be nice to have reasonable defaults later for the most of setters.
Otherwise this API will be really complicated to use externally